src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/security-recommendations/[type]/page.tsx

```diff
@@ -56,6 +57,10 @@ export default async function SecurityRecommendations({
         latestScanData: scanData,
       }}
       isEligibleForPremium={isEligibleForPremium(countryCode)}
+      enabledFeatureFlags={await getEnabledFeatureFlags({
```

Comment:
Somewhat unrelated to this PR: We should consider removing the deprecation notice from `getEnabledFeatureFlags`. We’ve kept adding and using feature flags.

Reply:
Might as well do that now, good point: 861a87870.